package.json

```diff
@@ -69,7 +69,7 @@
     "lcov-result-merger": "^1.2.0",
     "mocha": "^3.0.0",
     "mocha-lcov-reporter": "^1.2.0",
-    "nock": "^8.0.0",
+    "nock": "^9.0.4",
```

Comment:
Nock could not be updated until now, because they dropped 0.10 and 0.12 support in their new versions. See https://github.com/apiaryio/dredd/issues/660.